src/compositions/useWorkPoolLastPolled.ts

```diff
@@ -0,0 +1,65 @@
+import { SubscriptionOptions, UseSubscription, useNow, useSubscriptionWithDependencies } from '@prefecthq/vue-compositions'
+import max from 'date-fns/max'
+import { ComputedRef, MaybeRef, computed, ref } from 'vue'
+import { useWorkspaceApi } from '@/compositions/useWorkspaceApi'
+import { WorkPoolWorkersFilter } from '@/models'
+import { WorkspaceWorkPoolWorkersApi } from '@/services'
+import { formatDateTimeRelative } from '@/utilities'
+
+export type UseWorkPoolLastPolled = {
+  subscription: UseSubscription<WorkspaceWorkPoolWorkersApi['getWorkers']>,
+  lastPolled: ComputedRef<string | undefined>,
+}
+
+export function useWorkPoolLastPolled(
+  workPoolName: MaybeRef<string>,
+  filter?: MaybeRef<WorkPoolWorkersFilter | undefined>,
+  options?: SubscriptionOptions,
+): UseWorkPoolLastPolled {
+  const api = useWorkspaceApi()
+  const { now } = useNow({ interval: 1000 })
+
+  const workPoolNameRef = ref(workPoolName)
+  const filterRef = ref(filter)
+
+  const finalFilters = computed<[string, WorkPoolWorkersFilter] | null>(() => {
+    if (!filterRef.value) {
```

Comment:
Don't you still need to return the workPoolName even if there are no extra filters?